### PR TITLE
[CodingStyle] Improve unspread array

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: rules/coding-style/tests/Rector/ClassMethod/UnSpreadOperatorRector/Fixture/handle_multiple_items_to_array.php.inc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+-----
+<?php
+
+namespace Rector\CodingStyle\Tests\Rector\ClassMethod\UnSpreadOperatorRector\Fixture;
+
+final class HandleMultipleItemsToArray
+{
+    public function go()
+    {
+        $this->run('John', [1, 2, 3, 4, 5]);
```
Comment (TomasVotruba) [inline rules/coding-style/tests/Rector/ClassMethod/UnSpreadOperatorRector/Fixture/handle_multiple_items_to_array.php.inc:27]: @samsonasik Hey, just FYI, I had to update this rules to group items to an array like this.
It broke some code on Symplify.

File: rules/coding-style/src/Rector/ClassMethod/UnSpreadOperatorRector.php [hunk 2]
```diff
@@ -87,135 +84,70 @@ public function refactor(Node $node): ?Node
...
+            $spreadParam->type = new Identifier('array');
         }
 
         return $classMethod;
     }
 
-    /**
-     * @param ClassMethod|MethodCall $node
-     */
-    private function isInVendor(Node $node): bool
```
Comment (TomasVotruba) [inline rules/coding-style/src/Rector/ClassMethod/UnSpreadOperatorRector.php:163]: @samsonasik Just a tip for next code. The `/vendor` check can be done better by using `NodeRepository`. It contains all the nodes from parsed project. So what is not there, is in vendor :)

File: rules/coding-style/tests/Rector/ClassMethod/UnSpreadOperatorRector/Fixture/handle_multiple_items_to_array.php.inc (hunk 1 above)
Comment (samsonasik) [reply rules/coding-style/tests/Rector/ClassMethod/UnSpreadOperatorRector/Fixture/handle_multiple_items_to_array.php.inc:27]: Thank you

File: rules/coding-style/src/Rector/ClassMethod/UnSpreadOperatorRector.php (hunk 2 above)
Comment (samsonasik) [reply rules/coding-style/src/Rector/ClassMethod/UnSpreadOperatorRector.php:163]: Thank you

**Pull request merged**